AnkiDroid/src/main/java/com/ichi2/anki/multimedia/MultimediaImageFragment.kt

```diff
@@ -422,9 +425,10 @@ class MultimediaImageFragment : MultimediaFragment(R.layout.fragment_multimedia_
         imageFileSize.text = file.toHumanReadableSize()
     }
 
-    private fun showLargeFileCropDialog(length: Float) {
-        val decimalFormat = DecimalFormat(".00")
-        val size = decimalFormat.format(length.toDouble())
+    private fun showLargeFileCropDialog(length: Long) {
+        val numberFormat = NumberFormat.getInstance()
+        // length is in bits, other elements have MB, convert to MB
+        val size = numberFormat.format(length / 1000000)
```

Comment:
Otherwise, you use integer division

```suggestion
        val size = numberFormat.format(length / 1000000.0)
```

Reply:
My javascript-majority tech skills are showing
Nice catch